autoload/fzf/vim.vim

```diff
@@ -40,7 +43,10 @@ if s:is_win
   if has('nvim')
     let s:bin.preview = split(system('for %A in ("'.s:bin.preview.'") do @echo %~sA'), "\n")[0]
   else
-    let s:bin.preview = fnamemodify(s:bin.preview, ':8')
+    let preview_path = s:is_wsl_bash
+      \ ? substitute(s:bin.preview, '^\([A-Z]\):', '/mnt/\L\1', '')
+      \ : fnamemodify(s:bin.preview, ':8')
+    let s:bin.preview = substitute(preview_path, '\', '/', 'g')
```

Comment:
By doing this, we're applying this additional `substitute` call even when `s:is_wsl_bash` is false. Is this intentional? Have you tested this with non-WSL setup?

Reply:
It is intentional. 
When I tried on my `bash`-inside-git-installation-folder Windows environment, I had an error because of the backwards slashes.
The behavior you're describing is the intentional modification I wrote after that test.

Reply:
I'm leaving the `Resolve` button for you to click if you are OK with my answer :)

Reply:
Thanks for the explanation. I don't have a Windows PC to test this right now, so I'm just going to merge this and see if any Windows user complains :)